onnx/defs/nn/defs.cc

```diff
@@ -706,7 +709,9 @@ output_shape can also be explicitly specified in which case pads values are auto
         "height and width of the kernel, and M is the number "
         "of feature maps. For more than 2 dimensions, the "
         "weight shape will be (C x M x k1 x k2 x ... x kn), "
-        "where (k1 x k2 x ... x kn) is the dimension of the kernel",
+        "where (k1 x k2 x ... x kn) is the dimension of the kernel. "
```

Comment:
In convtranspose case, weight shape should be C/group x M x ...

Reply:
I would actually prefer that we rename C and M. The naming is confusing. In the case of conv transpose, num_output_channels = W.shape[1]/group. W.shape[1] is actually a multiplier for num_output_channels. W.shape[0] corresponds to num_input_channels (X.shape[1]). 
If we keep the same names, the spec should be C x M/group.

Reply:
Yes, this should be [C, M/group, spatial, spatial, ...]

Reply:
I would prefer to keep the names. But I am open to the new names, if you can think of any better name.

Reply:
Sure, lets keep [C, M/group, spatial, spatial, ...]